README.md

```diff
@@ -20,6 +20,7 @@ provisioning VMs, and [OpenShift Origin](https://github.com/openshift/origin) fo
   - [Deploying a sample application](#deploying-a-sample-application)
   - [Reusing the Docker daemon](#reusing-the-docker-daemon)
 - [Documentation](#documentation)
+- [Using Minishift with an HTTP/HTTPS Proxy](#using-minishift-with-an-proxy)
```

Comment:
It might be better to move this section below Starting Minishift section, wdyt? The user will need to use these flags with Minishift start.